backend/src/main/java/com/redhat/cloud/notifications/templates/EmailTemplateFactory.java

```diff
@@ -24,6 +24,10 @@ public EmailTemplate get(String bundle, String application) {
             if (application.equalsIgnoreCase("advisor")) {
                 return new AdvisorOpenshift();
             }
+        } else if (bundle.equals("application-services")) {
+            if (application.equals("rhosak")) {
```

Comment:
I don't know why we used `equalsIgnoreCase` for other templates but could you please do the same here? I'll have a look later to see if this `equalsIgnoreCase` can be replaced with `equals`.

Reply:
@theute confirmed that the name is `rhosak` that's why I used equal but no harm in ignoring case.

Reply:
Changed to ignoring cases for now.